### PR TITLE
Update some informations in manifest.json

### PR DESCRIPTION
**Pull request by realTiim**
## Type of change

- [ ] Add new directory for a server
- [x] Update informations of the server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x) I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/slown/manifest.json [hunk 1]
```diff
@@ -13,17 +13,14 @@
...
     "web_shop": "https://slown.tebex.io",
     "web_support": "https://www.slown.eu/forum/index.php?route=/forum/view/8-allgemeiner-support",
-    "twitter": "SlownEu",
-    "tiktok": "slowneu",
+    "twitter": "slownserver",
+    "tiktok": "slownserver",
     "discord": "https://discord.com/invite/QKncfGKFPf",
-    "youtube": "https://www.youtube.com/channel/UCx-reISgBwzM6rgATHz97vg",
+    "youtube": "https://www.youtube.com/channel/UCrVHevI55CrhfT8nl8vBMNw",
+    "instagram": "https://www.instagram.com/slownserver/",
```
Comment (JakobBruening) [inline minecraft_servers/slown/manifest.json:20]: Remove "/" at the end :)

**Comment by realTiim** [comment]
> _Just as an information_: Your server slown.eu **could be offline**. In general, we only accept pull requests from servers, **that are online**. Please change this, otherwise we cannot review your server correctly and have to deny the pull request.
> 
> If your server is currently online, then our api returned a wrong status, we will have a look at it :)
> 
> Reference: [API URL (https://api.mcsrvstat.us/2/slown.eu)](https://api.mcsrvstat.us/2/slown.eu)

the minecraft server is currently online.
ingame there are also some players online and you are welcome to take a look

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please fix the formatting and add a space after the colon:
e.g. ``"web":"https://www.slown.eu"`` -> ``"web": "https://www.slown.eu"`` (as before)

**Review by JakobBruening — APPROVED** [review]
Thank you!

**Pull request merged**